lib/hexpm_web/templates/email/package_published.html.eex

```diff
@@ -0,0 +1,6 @@
+<p><%= @username %> has recently published package <%= @package %>.</p>
```

Comment:
In this case `@username` will always be your own username. Unless we intend to send this to all owners I think we should change the text a little.

Since we do this for security reasons maybe we should give actions you can take if it wasn't you who published. For example, this is what Dropbox sends when it detects a login from a new device:

<img width="604" alt="Screen Shot 2019-04-01 at 14 38 47" src="https://user-images.githubusercontent.com/316890/55328188-12dae200-548c-11e9-8dd8-5baac38d9035.png">

I suggest:

You have recently published package <%= @package %> v<%= @version %>. If this wasn't done by you should reset your account: <link to reset page> and revert or retire the version:

For mix:
```
cd @package; mix hex.publish --revert @version
# or
mix hex.retire @package @version security --message "Not published by owners"
```

For rebar3:
```
cd @package; rebar3 hex publish --revert @version
# or
rebar3 hex retire --pkg @package --vsn @version --reason security --message "Not published by owners"
```

WDYT?

Reply:
sounds good to me 👍